src/tests/ftest/io/ior_small.py

```diff
@@ -48,4 +48,30 @@ def test_ior_small(self):
 
         :avocado: tags=all,daosio,small,pr,hw,iorsmall
         """
+        flags = self.params.get("ior_flags", '/run/ior/iorflags/*')
+        apis = self.params.get("ior_api", '/run/ior/iorflags/*')
+        transfer_block_size = self.params.get("transfer_block_size",
+                                              '/run/ior/iorflags/*')
+        obj_class = self.params.get("obj_class", '/run/ior/iorflags/*')
+
+        # run tests for different variants
+        for oclass in obj_class:
+            self.ior_cmd.daos_oclass.update(oclass)
+            for api in apis:
+                self.ior_cmd.api.update(api)
+                self.ior_cmd.flags.update(flags[0])
```

Comment:
If you're going to make another commit, you can move line #62 before #58 and mention in the comment that it's for ssf.